01week/datatypes.js

```diff
@@ -0,0 +1,98 @@
+// Write a JavaScript program to display the current day and time.
+
+const currentDayandTime =()=> {
+
+const currentDate = new Date();
+const day = currentDate.getDate();
+const month = currentDate.getMonth();
+const year = currentDate.getFullYear();
+
+const hours = currentDate.getHours();
+const minutes = currentDate.getMinutes();
+const seconds = currentDate.getSeconds();
+
+const time = currentDate.getUTCDate();
+
+return(month + "/" + day + "/" + year + "   " + "Time:" + hours +":" + minutes +":" + seconds );
+}
+
+console.log(currentDayandTime());
+
+
+// Write a JavaScript program to convert a number to a string.
+
+const numbertoString = (num1) => {
+  const s = num1.toString();
+
+  return(s);
```

Comment:
the parentheses are unnecessary and make it more difficult to read. Just `return num1.toString();`